.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
question: what is this for?

Reply:
I think we should let them choose to not use templates sometimes - wont' block the PR tho